addons/flux-new-service/apps/dev/<service_name>/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/@{{ service_name }}
+  # - app-SealedSecret-env.yaml
```

Comment:
Is this something we handle in a second step, once the sealed secret has been possibly created, correct? Does it mean that the app deployment will fail till the selaed secret is created, correct?